src/components/cylc/tree/deltas.js

```diff
@@ -49,7 +49,12 @@ function applyDeltasPruned (pruned, tree) {
   Object.keys(PRUNED).forEach(prunedKey => {
     if (pruned[prunedKey]) {
       for (const id of pruned[prunedKey]) {
-        tree[PRUNED[prunedKey]](id)
+        try {
+          tree[PRUNED[prunedKey]](id)
+        } catch (error) {
+          // eslint-disable-next-line no-console
+          console.error('Error applying pruned-delta, will continue processing the remaining data', error, id)
```

Comment:
I wouldn't want people to miss an important error like this. Perhaps we should make this a user facing error en that dismissible status bar thinggy?

Reply:
Bumped to a comment in #614